manifests/install.pp

```diff
@@ -93,18 +93,26 @@
       # SCL is only valid in the RedHat family. If RHEL, package must be
       # enabled using the subscription manager outside of puppet. If CentOS,
       # the centos-release-SCL will install the repository.
-      $install_scl_repo_package = $::operatingsystem ? {
-        'CentOS' => 'present',
-        default  => 'absent',
-      }
+      if $python::manage_scl {
+        $install_scl_repo_package = $::operatingsystem ? {
```

Comment:
the topscope fact is deprecated. could you migrate it to `$facts['os']['name']`?

Reply:
Done.